lib/ansible/modules/cloud/docker/docker_login.py

```diff
@@ -165,6 +164,10 @@ def __init__(self, client, results):
         self.reauthorize = parameters.get('reauthorize')
         self.config_path = parameters.get('config_path')
 
+        # As far as I can tell, email is not supported by credential helpers at all.
+        if self.email:
+            client.module.deprecate("The email parameter is deprecated and presently does nothing", "2.10")
```

Comment:
```suggestion
            client.module.deprecate("The email parameter is deprecated and presently does nothing", "2.13")
```
The Ansible deprecation cycle is always (at least) 4 versions, so if this is merged for 2.9, the option will be removed in 2.13.

Reply:
understood, I'll update.